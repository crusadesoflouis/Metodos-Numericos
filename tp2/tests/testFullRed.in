reduced/s1/1.pgm, 1,
reduced/s1/2.pgm, 1,
reduced/s1/3.pgm, 1,
reduced/s1/4.pgm, 1,
reduced/s1/5.pgm, 1,
reduced/s1/6.pgm, 1,
reduced/s1/7.pgm, 1,
reduced/s1/8.pgm, 1,
reduced/s1/9.pgm, 1,
reduced/s1/10.pgm, 1,
reduced/s2/1.pgm, 2,
reduced/s2/2.pgm, 2,
reduced/s2/3.pgm, 2,
reduced/s2/4.pgm, 2,
reduced/s2/5.pgm, 2,
reduced/s2/6.pgm, 2,
reduced/s2/7.pgm, 2,
reduced/s2/8.pgm, 2,
reduced/s2/9.pgm, 2,
reduced/s2/10.pgm, 2,
reduced/s3/1.pgm, 3,
reduced/s3/2.pgm, 3,
reduced/s3/3.pgm, 3,
reduced/s3/4.pgm, 3,
reduced/s3/5.pgm, 3,
reduced/s3/6.pgm, 3,
reduced/s3/7.pgm, 3,
reduced/s3/8.pgm, 3,
reduced/s3/9.pgm, 3,
reduced/s3/10.pgm, 3,
reduced/s4/1.pgm, 4,
reduced/s4/2.pgm, 4,
reduced/s4/3.pgm, 4,
reduced/s4/4.pgm, 4,
reduced/s4/5.pgm, 4,
reduced/s4/6.pgm, 4,
reduced/s4/7.pgm, 4,
reduced/s4/8.pgm, 4,
reduced/s4/9.pgm, 4,
reduced/s4/10.pgm, 4,
reduced/s5/1.pgm, 5,
reduced/s5/2.pgm, 5,
reduced/s5/3.pgm, 5,
reduced/s5/4.pgm, 5,
reduced/s5/5.pgm, 5,
reduced/s5/6.pgm, 5,
reduced/s5/7.pgm, 5,
reduced/s5/8.pgm, 5,
reduced/s5/9.pgm, 5,
reduced/s5/10.pgm, 5,
reduced/s6/1.pgm, 6,
reduced/s6/2.pgm, 6,
reduced/s6/3.pgm, 6,
reduced/s6/4.pgm, 6,
reduced/s6/5.pgm, 6,
reduced/s6/6.pgm, 6,
reduced/s6/7.pgm, 6,
reduced/s6/8.pgm, 6,
reduced/s6/9.pgm, 6,
reduced/s6/10.pgm, 6,
reduced/s7/1.pgm, 7,
reduced/s7/2.pgm, 7,
reduced/s7/3.pgm, 7,
reduced/s7/4.pgm, 7,
reduced/s7/5.pgm, 7,
reduced/s7/6.pgm, 7,
reduced/s7/7.pgm, 7,
reduced/s7/8.pgm, 7,
reduced/s7/9.pgm, 7,
reduced/s7/10.pgm, 7,
reduced/s8/1.pgm, 8,
reduced/s8/2.pgm, 8,
reduced/s8/3.pgm, 8,
reduced/s8/4.pgm, 8,
reduced/s8/5.pgm, 8,
reduced/s8/6.pgm, 8,
reduced/s8/7.pgm, 8,
reduced/s8/8.pgm, 8,
reduced/s8/9.pgm, 8,
reduced/s8/10.pgm, 8,
reduced/s9/1.pgm, 9,
reduced/s9/2.pgm, 9,
reduced/s9/3.pgm, 9,
reduced/s9/4.pgm, 9,
reduced/s9/5.pgm, 9,
reduced/s9/6.pgm, 9,
reduced/s9/7.pgm, 9,
reduced/s9/8.pgm, 9,
reduced/s9/9.pgm, 9,
reduced/s9/10.pgm, 9,
reduced/s10/1.pgm, 10,
reduced/s10/2.pgm, 10,
reduced/s10/3.pgm, 10,
reduced/s10/4.pgm, 10,
reduced/s10/5.pgm, 10,
reduced/s10/6.pgm, 10,
reduced/s10/7.pgm, 10,
reduced/s10/8.pgm, 10,
reduced/s10/9.pgm, 10,
reduced/s10/10.pgm, 10,
reduced/s11/1.pgm, 11,
reduced/s11/2.pgm, 11,
reduced/s11/3.pgm, 11,
reduced/s11/4.pgm, 11,
reduced/s11/5.pgm, 11,
reduced/s11/6.pgm, 11,
reduced/s11/7.pgm, 11,
reduced/s11/8.pgm, 11,
reduced/s11/9.pgm, 11,
reduced/s11/10.pgm, 11,
reduced/s12/1.pgm, 12,
reduced/s12/2.pgm, 12,
reduced/s12/3.pgm, 12,
reduced/s12/4.pgm, 12,
reduced/s12/5.pgm, 12,
reduced/s12/6.pgm, 12,
reduced/s12/7.pgm, 12,
reduced/s12/8.pgm, 12,
reduced/s12/9.pgm, 12,
reduced/s12/10.pgm, 12,
reduced/s13/1.pgm, 13,
reduced/s13/2.pgm, 13,
reduced/s13/3.pgm, 13,
reduced/s13/4.pgm, 13,
reduced/s13/5.pgm, 13,
reduced/s13/6.pgm, 13,
reduced/s13/7.pgm, 13,
reduced/s13/8.pgm, 13,
reduced/s13/9.pgm, 13,
reduced/s13/10.pgm, 13,
reduced/s14/1.pgm, 14,
reduced/s14/2.pgm, 14,
reduced/s14/3.pgm, 14,
reduced/s14/4.pgm, 14,
reduced/s14/5.pgm, 14,
reduced/s14/6.pgm, 14,
reduced/s14/7.pgm, 14,
reduced/s14/8.pgm, 14,
reduced/s14/9.pgm, 14,
reduced/s14/10.pgm, 14,
reduced/s15/1.pgm, 15,
reduced/s15/2.pgm, 15,
reduced/s15/3.pgm, 15,
reduced/s15/4.pgm, 15,
reduced/s15/5.pgm, 15,
reduced/s15/6.pgm, 15,
reduced/s15/7.pgm, 15,
reduced/s15/8.pgm, 15,
reduced/s15/9.pgm, 15,
reduced/s15/10.pgm, 15,
reduced/s16/1.pgm, 16,
reduced/s16/2.pgm, 16,
reduced/s16/3.pgm, 16,
reduced/s16/4.pgm, 16,
reduced/s16/5.pgm, 16,
reduced/s16/6.pgm, 16,
reduced/s16/7.pgm, 16,
reduced/s16/8.pgm, 16,
reduced/s16/9.pgm, 16,
reduced/s16/10.pgm, 16,
reduced/s17/1.pgm, 17,
reduced/s17/2.pgm, 17,
reduced/s17/3.pgm, 17,
reduced/s17/4.pgm, 17,
reduced/s17/5.pgm, 17,
reduced/s17/6.pgm, 17,
reduced/s17/7.pgm, 17,
reduced/s17/8.pgm, 17,
reduced/s17/9.pgm, 17,
reduced/s17/10.pgm, 17,
reduced/s18/1.pgm, 18,
reduced/s18/2.pgm, 18,
reduced/s18/3.pgm, 18,
reduced/s18/4.pgm, 18,
reduced/s18/5.pgm, 18,
reduced/s18/6.pgm, 18,
reduced/s18/7.pgm, 18,
reduced/s18/8.pgm, 18,
reduced/s18/9.pgm, 18,
reduced/s18/10.pgm, 18,
reduced/s19/1.pgm, 19,
reduced/s19/2.pgm, 19,
reduced/s19/3.pgm, 19,
reduced/s19/4.pgm, 19,
reduced/s19/5.pgm, 19,
reduced/s19/6.pgm, 19,
reduced/s19/7.pgm, 19,
reduced/s19/8.pgm, 19,
reduced/s19/9.pgm, 19,
reduced/s19/10.pgm, 19,
reduced/s20/1.pgm, 20,
reduced/s20/2.pgm, 20,
reduced/s20/3.pgm, 20,
reduced/s20/4.pgm, 20,
reduced/s20/5.pgm, 20,
reduced/s20/6.pgm, 20,
reduced/s20/7.pgm, 20,
reduced/s20/8.pgm, 20,
reduced/s20/9.pgm, 20,
reduced/s20/10.pgm, 20,
reduced/s21/1.pgm, 21,
reduced/s21/2.pgm, 21,
reduced/s21/3.pgm, 21,
reduced/s21/4.pgm, 21,
reduced/s21/5.pgm, 21,
reduced/s21/6.pgm, 21,
reduced/s21/7.pgm, 21,
reduced/s21/8.pgm, 21,
reduced/s21/9.pgm, 21,
reduced/s21/10.pgm, 21,
reduced/s22/1.pgm, 22,
reduced/s22/2.pgm, 22,
reduced/s22/3.pgm, 22,
reduced/s22/4.pgm, 22,
reduced/s22/5.pgm, 22,
reduced/s22/6.pgm, 22,
reduced/s22/7.pgm, 22,
reduced/s22/8.pgm, 22,
reduced/s22/9.pgm, 22,
reduced/s22/10.pgm, 22,
reduced/s23/1.pgm, 23,
reduced/s23/2.pgm, 23,
reduced/s23/3.pgm, 23,
reduced/s23/4.pgm, 23,
reduced/s23/5.pgm, 23,
reduced/s23/6.pgm, 23,
reduced/s23/7.pgm, 23,
reduced/s23/8.pgm, 23,
reduced/s23/9.pgm, 23,
reduced/s23/10.pgm, 23,
reduced/s24/1.pgm, 24,
reduced/s24/2.pgm, 24,
reduced/s24/3.pgm, 24,
reduced/s24/4.pgm, 24,
reduced/s24/5.pgm, 24,
reduced/s24/6.pgm, 24,
reduced/s24/7.pgm, 24,
reduced/s24/8.pgm, 24,
reduced/s24/9.pgm, 24,
reduced/s24/10.pgm, 24,
reduced/s25/1.pgm, 25,
reduced/s25/2.pgm, 25,
reduced/s25/3.pgm, 25,
reduced/s25/4.pgm, 25,
reduced/s25/5.pgm, 25,
reduced/s25/6.pgm, 25,
reduced/s25/7.pgm, 25,
reduced/s25/8.pgm, 25,
reduced/s25/9.pgm, 25,
reduced/s25/10.pgm, 25,
reduced/s26/1.pgm, 26,
reduced/s26/2.pgm, 26,
reduced/s26/3.pgm, 26,
reduced/s26/4.pgm, 26,
reduced/s26/5.pgm, 26,
reduced/s26/6.pgm, 26,
reduced/s26/7.pgm, 26,
reduced/s26/8.pgm, 26,
reduced/s26/9.pgm, 26,
reduced/s26/10.pgm, 26,
reduced/s27/1.pgm, 27,
reduced/s27/2.pgm, 27,
reduced/s27/3.pgm, 27,
reduced/s27/4.pgm, 27,
reduced/s27/5.pgm, 27,
reduced/s27/6.pgm, 27,
reduced/s27/7.pgm, 27,
reduced/s27/8.pgm, 27,
reduced/s27/9.pgm, 27,
reduced/s27/10.pgm, 27,
reduced/s28/1.pgm, 28,
reduced/s28/2.pgm, 28,
reduced/s28/3.pgm, 28,
reduced/s28/4.pgm, 28,
reduced/s28/5.pgm, 28,
reduced/s28/6.pgm, 28,
reduced/s28/7.pgm, 28,
reduced/s28/8.pgm, 28,
reduced/s28/9.pgm, 28,
reduced/s28/10.pgm, 28,
reduced/s29/1.pgm, 29,
reduced/s29/2.pgm, 29,
reduced/s29/3.pgm, 29,
reduced/s29/4.pgm, 29,
reduced/s29/5.pgm, 29,
reduced/s29/6.pgm, 29,
reduced/s29/7.pgm, 29,
reduced/s29/8.pgm, 29,
reduced/s29/9.pgm, 29,
reduced/s29/10.pgm, 29,
reduced/s30/1.pgm, 30,
reduced/s30/2.pgm, 30,
reduced/s30/3.pgm, 30,
reduced/s30/4.pgm, 30,
reduced/s30/5.pgm, 30,
reduced/s30/6.pgm, 30,
reduced/s30/7.pgm, 30,
reduced/s30/8.pgm, 30,
reduced/s30/9.pgm, 30,
reduced/s30/10.pgm, 30,
reduced/s31/1.pgm, 31,
reduced/s31/2.pgm, 31,
reduced/s31/3.pgm, 31,
reduced/s31/4.pgm, 31,
reduced/s31/5.pgm, 31,
reduced/s31/6.pgm, 31,
reduced/s31/7.pgm, 31,
reduced/s31/8.pgm, 31,
reduced/s31/9.pgm, 31,
reduced/s31/10.pgm, 31,
reduced/s32/1.pgm, 32,
reduced/s32/2.pgm, 32,
reduced/s32/3.pgm, 32,
reduced/s32/4.pgm, 32,
reduced/s32/5.pgm, 32,
reduced/s32/6.pgm, 32,
reduced/s32/7.pgm, 32,
reduced/s32/8.pgm, 32,
reduced/s32/9.pgm, 32,
reduced/s32/10.pgm, 32,
reduced/s33/1.pgm, 33,
reduced/s33/2.pgm, 33,
reduced/s33/3.pgm, 33,
reduced/s33/4.pgm, 33,
reduced/s33/5.pgm, 33,
reduced/s33/6.pgm, 33,
reduced/s33/7.pgm, 33,
reduced/s33/8.pgm, 33,
reduced/s33/9.pgm, 33,
reduced/s33/10.pgm, 33,
reduced/s34/1.pgm, 34,
reduced/s34/2.pgm, 34,
reduced/s34/3.pgm, 34,
reduced/s34/4.pgm, 34,
reduced/s34/5.pgm, 34,
reduced/s34/6.pgm, 34,
reduced/s34/7.pgm, 34,
reduced/s34/8.pgm, 34,
reduced/s34/9.pgm, 34,
reduced/s34/10.pgm, 34,
reduced/s35/1.pgm, 35,
reduced/s35/2.pgm, 35,
reduced/s35/3.pgm, 35,
reduced/s35/4.pgm, 35,
reduced/s35/5.pgm, 35,
reduced/s35/6.pgm, 35,
reduced/s35/7.pgm, 35,
reduced/s35/8.pgm, 35,
reduced/s35/9.pgm, 35,
reduced/s35/10.pgm, 35,
reduced/s36/1.pgm, 36,
reduced/s36/2.pgm, 36,
reduced/s36/3.pgm, 36,
reduced/s36/4.pgm, 36,
reduced/s36/5.pgm, 36,
reduced/s36/6.pgm, 36,
reduced/s36/7.pgm, 36,
reduced/s36/8.pgm, 36,
reduced/s36/9.pgm, 36,
reduced/s36/10.pgm, 36,
reduced/s37/1.pgm, 37,
reduced/s37/2.pgm, 37,
reduced/s37/3.pgm, 37,
reduced/s37/4.pgm, 37,
reduced/s37/5.pgm, 37,
reduced/s37/6.pgm, 37,
reduced/s37/7.pgm, 37,
reduced/s37/8.pgm, 37,
reduced/s37/9.pgm, 37,
reduced/s37/10.pgm, 37,
reduced/s38/1.pgm, 38,
reduced/s38/2.pgm, 38,
reduced/s38/3.pgm, 38,
reduced/s38/4.pgm, 38,
reduced/s38/5.pgm, 38,
reduced/s38/6.pgm, 38,
reduced/s38/7.pgm, 38,
reduced/s38/8.pgm, 38,
reduced/s38/9.pgm, 38,
reduced/s38/10.pgm, 38,
reduced/s39/1.pgm, 39,
reduced/s39/2.pgm, 39,
reduced/s39/3.pgm, 39,
reduced/s39/4.pgm, 39,
reduced/s39/5.pgm, 39,
reduced/s39/6.pgm, 39,
reduced/s39/7.pgm, 39,
reduced/s39/8.pgm, 39,
reduced/s39/9.pgm, 39,
reduced/s39/10.pgm, 39,
reduced/s40/1.pgm, 40,
reduced/s40/2.pgm, 40,
reduced/s40/3.pgm, 40,
reduced/s40/4.pgm, 40,
reduced/s40/5.pgm, 40,
reduced/s40/6.pgm, 40,
reduced/s40/7.pgm, 40,
reduced/s40/8.pgm, 40,
reduced/s40/9.pgm, 40,
reduced/s40/10.pgm, 40,
reduced/s41/1.pgm, 41,
reduced/s41/2.pgm, 41,
reduced/s41/3.pgm, 41,
reduced/s41/4.pgm, 41,
reduced/s41/5.pgm, 41,
reduced/s41/6.pgm, 41,
reduced/s41/7.pgm, 41,
reduced/s41/8.pgm, 41,
reduced/s41/9.pgm, 41,
reduced/s41/10.pgm, 41,