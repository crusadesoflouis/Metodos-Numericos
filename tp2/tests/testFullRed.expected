422.906700
361.552731
268.316560
235.708580
223.290532
189.496399
162.195107
151.590903
140.346310
133.104904
119.540182
116.963259
109.594195
103.132954
100.467828
